raiden/tests/smart_contracts/test_channel_manager.py

```diff
@@ -188,11 +198,20 @@ def test_reopen_channel(
     # now a single new channel can be opened
     # if channel with address is settled a new can be opened
     # old entry will be deleted when calling newChannel
-    netting_channel_address1_hex = tester_channelmanager.newChannel(
-        address1,
-        settle_timeout,
-        sender=privatekey0_raw,
-    )
+    with print_gas_used(tester_state, 'gas for newChannel when deleteChannel function called'):
```

Comment:
Is this print-debugging? I suppose you added it temporarily to check something and forgot to delete?